dist/rpm/os-autoinst.spec

```diff
@@ -59,7 +59,7 @@ Source0:        %{name}-%{version}.tar.xz
 %else
 %define spellcheck_requires %{nil}
 %endif
-%if 0%{?sle_version} < 150200 && !0%{?is_opensuse}
+%if 0%{?suse_version} < 1520 && !0%{?is_opensuse}
```

Comment:
15-SP2 should be already out of support, maybe we can get rid of this block entirely?